test/unit/cssTest.js

```diff
@@ -0,0 +1,268 @@
+const assert = require('assert')
+const path = require('path')
+const generateTestApp = require('../util/generateTestApp')
+const cleanupTestApp = require('../util/cleanupTestApp')
+const fork = require('child_process').fork
+const fs = require('fs')
+const fse = require('fs-extra')
+const klawsync = require('klaw-sync')
+
+// appDir
+const appDir = path.join(__dirname, '../', 'app', 'cssTest')
+
+// basic CSS files
+let cssDataArray = [
+  `body { 
+    height: 100%;
+  } 
+  h1 {
+    font-size: 10px;
+  }`,
+  `#selector { 
+    height: 100%;
+  } 
+  h1 {
+    font-size: 10px;
+  }`,
+  `.body { 
+    height: 100%;
+  } 
+  h1 {
+    font-size: 10px;
+  }`
+]
+
+// version package json file
+let packageJSON = `{
+ "version": "0.3.1",
+ "rooseveltConfig": {}
+}`
+
+// version num
+let verNumber = '0.3.1'
+
+// path to CSS static files
+let pathOfCSSStaticFilesArray = [
+  path.join(appDir, 'statics', 'css', 'a.css'),
+  path.join(appDir, 'statics', 'css', 'b.css'),
+  path.join(appDir, 'statics', 'css', 'c.css')
+]
+
+// path to CSS compiled files
+let pathOfCSSCompiledfilesArray = [
+  path.join(appDir, 'statics', '.build', 'css', 'a.css'),
+  path.join(appDir, 'statics', '.build', 'css', 'b.css'),
+  path.join(appDir, 'statics', '.build', 'css', 'c.css')
+]
+
+// function to generate the static files before each test
+const generateStaticFolder = () => {
+  // generate the folder for the static stuff first
+  fse.ensureDirSync(path.join(appDir, 'statics', 'css'))
+  // loop through the cssDataArry and write to each files in the paths of CSS Static Array
+  for (let x = 0; x < cssDataArray.length; x++) {
+    fs.writeFileSync(pathOfCSSStaticFilesArray[x], cssDataArray[x])
+  }
+}
+
+describe('CSS Section Tests', function () {
+// 1. Test to see that it compiles -check
+// 2. Test to see the whitelist compiles -check
+// 3. Test to see that changing the output name would in fact change the build output name and put compiled files in it -check
+// 4. Test to see if versionFile, if given a object, makes a versionFile CSS file - it exist, but version is not listed
+
+  // clean up the old test after completion
+  afterEach(function (done) {
+    cleanupTestApp(appDir, (err) => {
+      if (err) {
+        throw err
+      } else {
+        done()
+      }
+    })
+  })
+
+  it('should compile the static CSS files using roosevelt-less', function (done) {
+    // generate the static files and folder
+    generateStaticFolder()
+    // generate the app
+    generateTestApp({
+      appDir: appDir,
+      css: {
+        compiler: {
+          nodeModule: 'roosevelt-less',
+          params: {
+            cleanCSS: {
+              advanced: true,
+              aggressiveMerging: true
+            },
+            sourceMap: null
+          }
+        }
+      },
+      generateFolderStructure: true
+    }, 'initServer')
+
+    // fork the app
+    const testApp = fork(path.join(appDir, 'app.js'), ['-p'], {'stdio': ['pipe', 'pipe', 'pipe', 'ipc']})
```

Comment:
No need to pass `-p` or any other args to the app. By default it's in production mode.